src/server/models/module.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (sequelize, DataTypes) => {
+  const Module = sequelize.define('Module', {
+    name: DataTypes.STRING,
+    totalDuration: DataTypes.INTEGER
+  });
+
+  Module.associate = (models) => {
+    Module.hasMany(models.Challenge, {
```

Comment:
A `moduleId` foreign key on the challenge model; I think this satisfies our schema requirements